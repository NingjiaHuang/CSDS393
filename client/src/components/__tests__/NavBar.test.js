import {render, screen, cleanup} from '@testing-library/react';
import NavBar from '../NavBar/Navbar';

import renderer from 'react-test-renderer'
import '@testing-library/jest-dom/extend-expect';

afterEach(() =>{
    cleanup();
})


test('matches snapshot of navbar', () =>{
    const tree = renderer.create(<NavBar/>).toJSON();
    expect(tree).toMatchSnapshot();
})
