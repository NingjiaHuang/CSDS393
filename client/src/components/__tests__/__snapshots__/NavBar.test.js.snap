// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot of navbar 1`] = `
<nav
  className="NavbarItems"
>
  <h1
    className="navbar-logo"
  >
    PatCat
    <i
      className="fab fa-react"
    />
  </h1>
  <div
    className="menu-icon"
    onCilck={[Function]}
  >
    <i
      className="fas fa-bars"
    />
  </div>
  <ul
    className="nav-menu"
  >
    <li>
      <a
        className="nav-links"
        href="http://localhost:3000/register/breeder"
      >
        Breeder Register
      </a>
    </li>
    <li>
      <a
        className="nav-links"
        href="http://localhost:3000/login/breeder"
      >
        Breeder Login
      </a>
    </li>
    <li>
      <a
        className="nav-links"
        href="http://localhost:3000/register/parent"
      >
        Parent Register
      </a>
    </li>
    <li>
      <a
        className="nav-links"
        href="http://localhost:3000/login/parent"
      >
        Parent Login
      </a>
    </li>
    <li>
      <a
        className="nav-links"
        href="http://localhost:3000/login/admin"
      >
        Administrator Login
      </a>
    </li>
  </ul>
</nav>
`;
